scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1617,6 +1543,13 @@ def addDBVersion(version):
        spin_db_username = sys.argv[21]
        spin_db_password = sys.argv[22]
     migrate_data_flag = sys.argv[23]
+    
+    isd_commongate = sys.argv[24]
```

Comment:
@sudhakaropsmx rename "isd_commongate" to something more appropriate like - "isd_commongate_enabled" so it is easy to make out that it is a boolean type input

Reply:
removed commongate parameter